src/harness/testcases/WINNF_FT_S_EXZ_testcase.py

```diff
@@ -0,0 +1,293 @@
+    conditionalsN2 = {
+        'cbsdCategory': device_N2_2['cbsdCategory'],
+        'fccId': device_N2_2['fccId'],
+        'cbsdSerialNumber': device_N2_2['cbsdSerialNumber'],
+        'airInterface': device_N2_2['airInterface'],
+        'installationParam': device_N2_2['installationParam'],
+        'measCapability': device_N2_2['measCapability']
+    }
+    del device_N2_2['cbsdCategory']
+    del device_N2_2['airInterface']
+    del device_N2_2['installationParam']
+
+    self.assertEqual(device_N3_2['cbsdCategory'], 'B')
+    conditionalsN3 = {
+        'cbsdCategory': device_N3_2['cbsdCategory'],
+        'fccId': device_N3_2['fccId'],
+        'cbsdSerialNumber': device_N3_2['cbsdSerialNumber'],
+        'airInterface': device_N3_2['airInterface'],
+        'installationParam': device_N3_2['installationParam'],
+        'measCapability': device_N3_2['measCapability']
+    }
+    del device_N3_2['cbsdCategory']
+    del device_N3_2['airInterface']
+    del device_N3_2['installationParam']
+
+    self.assertEqual(device_N4_1['cbsdCategory'], 'B')
+    conditionalsN4 = {
+        'cbsdCategory': device_N4_1['cbsdCategory'],
+        'fccId': device_N4_1['fccId'],
+        'cbsdSerialNumber': device_N4_1['cbsdSerialNumber'],
+        'airInterface': device_N4_1['airInterface'],
+        'installationParam': device_N4_1['installationParam'],
+        'measCapability': device_N4_1['measCapability']
+    }
+    del device_N4_1['cbsdCategory']
+    del device_N4_1['airInterface']
+    del device_N4_1['installationParam']
+
+    # Create the actual config.
+    grants_N2 = [grant_N2_1, grant_N2_2]
+    grants_N3 = [grant_N3_1, grant_N3_2]
+    grants_N4 = [grant_N4_1, grant_N4_2]
+
+    devices_N2 = [device_N2_1, device_N2_2]
+    devices_N3 = [device_N3_1, device_N3_2]
+    devices_N4 = [device_N4_1, device_N4_2]
+
+    conditionals = {
+        'registrationData': [conditionalsN2, conditionalsN3, conditionalsN4]
+    }
+
+    config = {
+        'registrationRequestsN2' : devices_N2,
+        'registrationRequestsN3' : devices_N3,
+        'registrationRequestsN4' : devices_N4, 
+        'exclusionZoneRecords' : [exz_record_1, exz_record_2],
+        'grantRequestsN2' : grants_N2,
+        'grantRequestsN3' : grants_N3, 
+        'grantRequestsN4' : grants_N4,
+        'conditionalRegistrationData': conditionals
+      }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_EXZ_1_default_config)
+  def test_WINNF_FT_S_EXZ_1(self,config_filename):
+    """Injected Exclusion Zones
+
+    The response should be SUCCESS for CBSDs located outside 50meters of all 
+    Exclusion Zones.
+    The responseCode = 400 for CBSDs located within 50meters of all Exclusion Zones 
+    or inside Exclusion Zones. 
+    """
+    config = loadConfig(config_filename)
+
+    # Very light checking of the config file.
+    self.assertGreater(len(config['exclusionZoneRecords']) , 0, "Exclusion zone records are zero !!!")
```

Comment:
Please improve the English here, or remove the debug message since the default one will be clear enough.

Reply:
Removed   the debug message as suggested.